golang/examples/hello/client/main.go

```diff
@@ -21,27 +21,28 @@ package main
 // THE SOFTWARE.
 
 import (
-	"code.google.com/p/getopt"
 	"github.com/uber/tchannel/golang"
 	"golang.org/x/net/context"
+	"flag"
 	"os"
 	"time"
 )
 
 var log = tchannel.SimpleLogger
 
-var peerAddr = getopt.StringLong("peer", 'p', "", "Host and port of remote peer")
-var serviceName = getopt.StringLong("service", 's', "", "Name of service to invoke")
-var operationName = getopt.StringLong("operation", 'o', "", "Name of operation to invoke")
-var arg2 = getopt.StringLong("arg2", '2', "", "Input for arg2.  Curl-style, use @foo.txt to read from foo.txt")
-var arg3 = getopt.StringLong("arg3", '3', "", "Input for arg3.  Curl-style, use @foo.txt to read from foo.txt")
-var timeout = getopt.IntLong("timeout", 't', 30, "Timeout (in seconds)")
+var peerAddr = flag.String("peer", "localhost:10500", "Host and port of remote peer")
+var serviceName = flag.String("service", "TestService", "Name of service to invoke")
+var operationName = flag.String("operation", "echo", "Name of operation to invoke")
+var arg2 = flag.String("arg2", "hello", "Input for arg2.  Curl-style, use @foo.txt to read from foo.txt")
+var arg3 = flag.String("arg3", "world", "Input for arg3.  Curl-style, use @foo.txt to read from foo.txt")
+var timeout = flag.Int("timeout", 30, "Timeout (in seconds)")
 
 func asArgument(arg string) tchannel.Output {
 	if arg[0] == '@' {
 		f, err := os.Open(arg[1:])
 		if err != nil {
-			panic(err)
+			log.Errorf(err.Error())
+			os.Exit(-1)
```

Comment:
I tried that first, but tchannel.SimpleLogger doesn't mimic stdlib log (yet).  There's a few different problems within the  SimpleLogger. It's on my radar to take a look at.